contracts/test/crosschain/EntitlementChecker.t.sol

```diff
@@ -121,4 +135,26 @@ contract EntitlementCheckerTest is BaseSetup, IEntitlementCheckerBase {
     vm.expectRevert(EntitlementChecker_InsufficientNumberOfNodes.selector);
     entitlementChecker.getRandomNodes(26);
   }
+
+  // =============================================================
+  //                        Nodes by Operator
+  // =============================================================
+
+  function test_getNodesByOperator() external {
+    for (uint256 i = 0; i < operators.length; i++) {
+      uint256 totalNodes = 0;
```

Comment:
Can you rewrite this test case to catch any errors pointed out above?

Reply:
For this test case, I was thinking specifically, validate that these are the nodes registered by a specific operator. Do you think these changes would have caught the error that was fixed above where all nodes were included?